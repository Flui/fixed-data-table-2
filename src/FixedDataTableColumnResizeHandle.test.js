import React from 'react';
import FixedDataTableColumnResizeHandle from 'FixedDataTableColumnResizeHandle';
import ShallowRenderer from 'react-test-renderer/shallow';

it('renders correctly', () => {
  const props = {
    height: 100,
    knobHeight: 20,
    leftOffset: 0,
    visible: true,
  }

  const renderer = new ShallowRenderer();
  renderer.render(<FixedDataTableColumnResizeHandle {...props}></FixedDataTableColumnResizeHandle>);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});