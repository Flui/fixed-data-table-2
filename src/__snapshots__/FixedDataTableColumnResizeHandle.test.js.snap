// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="fixedDataTableColumnResizerLineLayout_main public_fixedDataTableColumnResizerLine_main"
  style={
    Object {
      "height": 100,
      "left": 0,
      "width": 0,
    }
  }
>
  <div
    className="fixedDataTableColumnResizerLineLayout_mouseArea"
    style={
      Object {
        "height": 100,
      }
    }
  />
</div>
`;
